Rename activate to sync

geekbeast opened the pull request:
Mostly for readability.

geekbeast committed:
Rename activate to sync

makosblade reviewed (approved):
Thoughts on calling it `syncUser` ?
I get that Principal API may imply that it's a user but otherwise I don't really know what's being sync'd by reading it

geekbeast committed:
Merge branch 'develop' into bugfix/LATTICE-2341-realtime-user-sync

geekbeast committed:
Fix path + clearer naming

geekbeast committed:
t Merge branch 'bugfix/LATTICE-2341-realtime-user-sync' of github.com:openlattice/api into bugfix/LATTICE-2341-realtime-user-sync

* 'bugfix/LATTICE-2341-realtime-user-sync' of github.com:openlattice/api:
  bugfixes
  entityId -> entityKeyId
  entityId -> entityKeyId
  this fixes it for real
  1 not set
  include all search API endpoints
  ugh
  fixing auth name error
  fixing auth name error
  swagger doesn't want relative references
  adding raw dataset and column endpoints
  LATTICE-2330 add openlattice doc
  getEntitySetsById, getEntitySetsByName
  LATTICE-2330 use post for loadLinkedEntitySetBreakdown
